src/api/export.ts

```diff
@@ -146,9 +146,10 @@ async function _export({
 
 		const r = await q.add(async () => {
 			const timeout_deferred = new Deferred();
+			
 			const the_timeout = setTimeout(() => {
 				timeout_deferred.reject(new Error(`Timed out waiting for ${url.href}`));
-			}, timeout);
+			}, timeout === false ? undefined : timeout);
```

Comment:
Shouldn't you just be not creating the timeout at all if `timeout === false`? I'd expect calling `setTimeout(..., undefined);` would have the same problem of it being called right away.